docs/sources/tempo/metrics-from-traces/metrics-queries/functions.md

```diff
@@ -335,3 +335,54 @@ This example means the attribute `resource.cluster` had too many values.
 ```
 { __meta_error="__too_many_values__", resource.cluster=<nil> }
 ```
+
+## Query hints and sampling
```

Comment:
This section is too general and doesn't exactly apply to metrics sampling per se.